src/components/container/SwellingCalendar/SwellingCalendar.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import Card from "../../presentational/Card"
+import Layout from "../../presentational/Layout"
+import SwellingCalendar from "./SwellingCalendar";
+import { SurveyResultDataParameters } from "../../../helpers/daily-data-providers";
+
+export default {
+    title: "Container/SwellingCalendar",
+    component: SwellingCalendar,
+    parameters: {
+        layout: 'fullscreen',
+    }
+} as ComponentMeta<typeof SwellingCalendar>;
```

Comment:
This is using the deprecated version of story setup.  It should be converted to using the newer pattern.   See the `CalendarDay` stories for an example.

https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/presentational/CalendarDay/CalendarDay.stories.tsx